### ISSUE TITLE
report time field autosort appears to do a text based sort instead of a time based sort (use time functions)

### ISSUE THREAD

**Issue by princealexi25** [eq-p99-tools/ninjalooter#10]
when clicking report time to sort, it sorts by day name, not by the whole element as a datetime data type thereby sorting to all friday's, then month, then day then xx:xx:xx then year

it looks like it is doing a text data type based sort vs a datetime data type based sort.

**Comment by princealexi25** [comment]
once you click the sort at the top of the column, the column is forever broken.

**Comment by rm-you** [comment]
Yeah that sounds likely, I don't recall customizing the sort function for any of the columns. Can look into it, being able to sort at all was just a "turn it on" thing in that ui element, but I'll look into why it's breaking too.
